packages/astro-rss/src/index.ts

```diff
@@ -69,6 +71,7 @@ function mapGlobResult(items: GlobResult): Promise<RSSFeedItem[]> {
 				title: frontmatter.title,
 				pubDate: frontmatter.pubDate,
 				description: frontmatter.description,
+				content: compiledContent(),
```

Comment:
Thanks @smithbm2316!

Looks like a test is failing because this approach is not compatible with MDX files which don’t provide a `compiledContent` getter. Not sure we have a good story that would let us resolve this just yet, but @bholmesdev looked at this recently so might be able to say if there’s a path forward here or if another approach is needed.

Reply:
Yes, apologies. I forgot to mention that this *definitely* is not compatible with MDX. The RSS integration currently *only* works with MDX files if you keep all of your data for each post inside the frontmatter at the moment correct? If the Astro team would consider this use case, I'd be very interested in helping @bholmesdev explore some options for how to resolve the MDX issues as well! Perhaps I can drop him a message in Discord and see if there's anything I can do to help with that. Thanks for the speedy comment and review @delucis! :smile:

Reply:
@smithbm2316 Yep, spent a pretty long time investigating options yesterday. Funny how timing works out 😅 Supporting compiled content in MDX is a separate issue that will need a couple months to get right. In the meantime, I'm considering a rework for the RSS integration that pulls from the production build of your site instead to get the compiled HTML body of your posts, similar to the sitemap integration. This _will_ bust dev server previews and SSR, but will unlock using post contents in your RSS feeds. I don't feel great about this solution but I'm happy to prototype by Monday.